dataproc/snippets/create_cluster_test.py

```diff
@@ -53,8 +57,21 @@ def test_cluster_create(capsys):
     # Wrapper function for client library function
     try:
         create_cluster.create_cluster(PROJECT_ID, REGION, CLUSTER_NAME)
+        out, _ = capsys.readouterr()
+        assert CLUSTER_NAME in out
+    except AlreadyExists:
+        request = dataproc.GetClusterRequest(project_id=PROJECT_ID, region=REGION, cluster_name=CLUSTER_NAME)
+        response = cluster_client.get_cluster(request=request)
+        assert response.status.state == dataproc.ClusterStatus.State(2)  # verify the cluster is in the RUNNING state
```

Comment:
Nit: Is there a `RUNNING` constant or something to avoid the magic number 2? If not, that's fine as well.

Reply:
turns out there is! Made that swap